behave/features/cla_fala/fala_end_to_end_tests.feature

```diff
@@ -59,3 +60,16 @@ Feature: FALA end to end tests
       | cy              | Welsh         | Dewch                  |
       | gd              | Scots Gaelic  | Lorg                   |
 
+
+  @fala-filtering-homepage
+  Scenario Outline: Applying filters on the homepage, filters correctly
```

Comment:
might be useful to have extra explanation that this scenario is specifically checking the filter works from the homepage.